src/test/scala/edu/berkeley/ce/rockslicing/RockSlicerSpec.scala

```diff
@@ -205,157 +204,212 @@ class RockSlicerSpec extends FunSuite {
   )
   val topRightEighthUnitCube = Block((0.0, 0.5, 0.5), boundingFaces20)
 
+ test("The two cubes should share one processor face") {
+   val leftCenter = (leftCube.centerX, leftCube.centerY, leftCube.centerZ)
+   val updatedRightCube = Block(leftCenter, rightCube.updateFaces(leftCenter))
+
+   val sharedFaces = RockSlicer.compareProcessorBlocks(leftCube, updatedRightCube)
+   assert(!sharedFaces.exists( face => math.abs(face.d) != 1.0))
+   assert(sharedFaces.length == 2)
+ }
+
+ test("The two blocks should share one processor face, but with distances reversed") {
+   val rightCenter = (rightCube.centerX, rightCube.centerY, rightCube.centerZ)
+   val updatedLeftCube = Block(rightCenter, leftCube.updateFaces(rightCenter))
+   val sharedFaces = RockSlicer.compareProcessorBlocks(rightCube, updatedLeftCube)
+   assert(!sharedFaces.exists( face => face.d != 0.0))
```

Comment:
Couldn't you simplify this to `assert(sharedFaces.forAll(_.d == 0.0)` or something along those lines?